frontend/packages/kubevirt-plugin/src/utils/index.ts

```diff
@@ -139,6 +140,9 @@ export const getResource = (
   const res: any = {
     // non-admin user cannot list namespaces (k8s wont return only namespaces available to user but 403 forbidden, ).
     // Instead we need to use ProjectModel which will return available projects (namespaces)
+    //
+    // FIXME: This is incorrect! `m.kind` is not unique. These model definitions should have `crd: true`, which will
+    // break this utility. We should be using `referenceForModel` and `crd: true` in our model definitions!
     kind: m.kind,
```

Comment:
@vojtechszocs FYI, this is a problem... (see my comment). It looks like we might be doing this throughout kubevirt, and it will cause weird bugs in different parts of the UI like https://bugzilla.redhat.com/show_bug.cgi?id=1768771 and the broken search page for NetworkAttachmentDefinition.

I'll probably open a bug to look at this, but the change likely won't be small :(